### PR TITLE
adding mongo db package.

### PR DESCRIPTION
**Pull request by TSteele27**

### CODE REVIEW

File: packages/mongodb/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+It expects the following config values:
+```
+config: {
+  mongodb: {
+		user,
+		password,
+		url,
+		databases: {
+			<database-name>: {
+				<collection-name>
```
Comment (BReul) [inline packages/mongodb/README.md]:
```suggestion
				<collection-name>: '<collection-name>'
```

File: packages/mongodb/db/index.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  try {
+    const client = await MongoClient.connect(url, {
+      auth: user ? {
+        password,
+        user
+      } : undefined,
+      poolSize,
+      useUnifiedTopology: true
+    })
+    process.env.UV_THREADPOOL_SIZE = Number(poolSize)
```
Comment (BReul) [inline packages/mongodb/db/index.js:30]: this was a quick patch we did for a LM service.  Not sure if this should be handled a bit more elegantly

Comment (rajeshsusai) [reply packages/mongodb/db/index.js:30]: We may not want to include that, placing a number that exceeds the node UV_THREADPOOL_SIZE could become problematic

Comment (TSteele27) [reply packages/mongodb/db/index.js:30]: do you remember what this was for exactly? maybe we can just remove it all together.

**Pull request closed without merging**